README.md

```diff
@@ -77,6 +77,35 @@ You can add a `.sonatype-config` file (`.sonatype-config.yaml` and `.sonatype-co
 
 `Stage` will default to `develop` which generates a report accessible only through the link generated by the evaluation dialog box.  The other stages represent state and are reachable through IQ reports page. 
 
+### Development Dependencies
+
+Some formats give us the ability to filter out the development dependencies on your project. By default we include them, because we want to give you the absolute best insight we can into your project, but in case you determine they are mostly noise, you can exclude them, IF the format allows us to:
+
+| Format | Supported |
+| --- | -- |
+| Golang | |
+| Maven | :heavy_check_mark: |
+| Gradle | :heavy_check_mark: |
+| RubyGems | |
+| Pip (Requirements.txt) | |
+| Poetry (poetry.lock) | :heavy_check_mark: |
+| Yarn | |
+| NPM | :heavy_check_mark: |
+| R | |
+| Cargo | |
+| Conan | |
+
+To switch to excluding dev dependencies, create a `.sonatype-config` file in your repo, akin to the following:
+
+```yml
+---
+    application:
+        # true or false!
+        IncludeDev: false
```

Comment:
Does the capitalization matter here, or does it just "handle it"? `IncludeDev` vs `includeDev`

Reply:
I proper cased it if only because we did that with the other properties in all the Go projects. CONSISTENCY.